projects/keycloak/JweFuzzer.java

```diff
@@ -63,7 +64,12 @@ public static void fuzzerTestOneInput(FuzzedDataProvider data) throws Exception
       String enc = data.pickValue(encs);
 
       // Create JweKeyStorage
-      Key key = KeyUtils.loadSecretKey(data.consumeBytes(32), "HmacSHA256");
+      byte[] keyBytes = data.consumeBytes(32);
+      if (keyBytes.length == 0) {
```

Comment:
Why don't we return if there are no more bytes? What is the purpose of proceeding?

Reply:
The creation of the key with random bytes are less important, the more important fuzzing point is the parse with key with random (or empty input) in the later process. So it should still continue with default key even if there are no more bytes and fuzz the behaviour of the later process.

Reply:
What does the fuzzer get out of progressing if there are no more bytes at this point? How can it randomize anything else?